addons/web_editor/static/src/js/common/utils.js

```diff
@@ -435,6 +435,40 @@ function _addBackgroundImageAttributes(...newAttributes) {
 function _isBackgroundImageAttribute(attribute) {
     return BACKGROUND_IMAGE_ATTRIBUTES.has(attribute);
 }
+/**
+ * This function handles text wrapping positions for an image based on the given options.
+ * It adds or removes CSS classes to adjust the image alignment and display.
+ * The 'options' parameter determines the specific options to be taken: 'inline', 'wrap-text', or 'break-text'.
+ *
+ * @param {HTMLElement} image
+ * @param {string} options
+ */
+function updateImageWrapping(image, options) {
+    if (options === 'inline') {
+        image.classList.remove('float-start', 'float-end', 'mx-auto', 'ms-auto', 'me-auto', 'd-flex', 'd-block');
+    } else if (options === 'wrap-text') {
+        const positionClasses = ['ms-auto', 'float-end'];
+        positionClasses.forEach(classToCheck => {
+            if (!image.classList.contains(classToCheck)) {
+                image.classList.remove('mx-auto', 'd-block', 'ms-auto');
+                image.classList.add('float-start','me-auto');
+            } else {
+                image.classList.remove('d-block', 'me-auto');
+                image.classList.add('float-end', 'ms-auto');
+            }
+        });
+    } else if (options === 'break-text') {
+        if (image.classList.contains('float-start')) {
+            image.classList.remove('float-start', 'me-auto');
+            image.classList.add('d-block', 'me-auto');
```

Comment:
Yes, that was a mistake because in previous code i have just add `d-flex` and `float-start` in place of `d-block` and `me-auto` in that code i was removing `me-auto` but after some changes i just forgot to remove it.